components/Banner/examples/supplemental/MDCBannerView.h

```diff
@@ -14,19 +14,33 @@
 
 #import <UIKit/UIKit.h>
 
-@class MDCButton;
+#import "MaterialButtons.h"
 
 __attribute__((objc_subclassing_restricted)) @interface MDCBannerView : UIView
 
-@property(nonatomic, copy, nonnull) NSString *text;
+/**
+ A view that displays the text on a @c MDCBannerView.
+ @discussion These properties of @c textLabel can be used to configure the text shown on @c MDCBannerView.
```

Comment:
The `@discussion` markers aren't necessary here and throughout. Delete them and place an empty line between the introductory line and the discussion text, as per our code's existing convention.

E.g.:

```
A view that displays the text on a @c MDCBannerView.

These properties of @c textLabel can be used to configure the text shown on @c MDCBannerView.
```

Reply:
Removed `@discussion` and addressed in 5ed33d9c7280f7d2798dab0b2aad4651983c8238 to match our code convention.